### DISCUSSION TITLE
Getting Started with Github Marketplace

### DISCUSSION THREAD

**Discussion by RorryB** [community/community#146297] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

I'm new to the idea of Github Marketplace.  I was wondering if my product has a place there or if thats not really what it's for.  I have developed a new type of deep learning optimizer.  You could compare it to Adam vs SGD vs RMSProp vs my new one.  Its installable with a pip install but not open source.

**Comment by mecodeatlas** [comment]
Thanks for posting in the GitHub Community, @RorryB!

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the [GitHub product Discussions](https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/about-discussions). This question should be in the General category. I've gone ahead and moved it for you. Good luck!
